src/io/flutter/utils/FlutterModuleUtils.java

```diff
@@ -172,8 +214,23 @@ public static boolean usesFlutter(@NotNull Module module) {
     return CollectionUtils.filter(getModules(project), m -> isFlutterModule(m) || usesFlutter(m));
   }
 
+  public static void convertFromDeprecatedModuleType(@NotNull Project project) {
+    for (Module module : getModules(project)) {
+      if (isDeprecatedFlutterModuleType(module)) {
+        setFlutterModuleType(module);
```

Comment:
When we do this, we should show a toast (`FlutterMessages.showInfo`) letting the user know - it'll help take the edge off the unknown module type toast that will also likely appear.